runtime/oti/j9nonbuilder.h

```diff
@@ -5002,7 +5002,7 @@ typedef struct J9VMContinuation {
 	struct J9JITGPRSpillArea jitGPRs;
 	struct J9I2JState i2jState;
 	struct J9VMEntryLocalStorage* oldEntryLocalStorage;
-	struct J9VMThread* carrierThread;
+	UDATA carrierThreadAndConcurrentlyScanned;
```

Comment:
I think you can declare here as  volatile UDATA, and lines like this should compile without explicit casting:

volatile uintptr_t *localAddr = &continuation->carrierThreadAndConcurrentlyScanned;

I'm still thinking about the proper name. This 'and' is confusing, since it may imply to someone as a logical 'and' while it's really means a bit-wise 'structure'. We could replace it with '_' or completely remove it... or do you have any other suggestion?

Reply:
I think '_' is better.